### ISSUE TITLE
WCAG violations in Cypress testing

### ISSUE THREAD

**Issue by olemartinorg** [Altinn/app-frontend-react#1268]
### Description of the bug

After #1267, there are now multiple known WCAG violations, all noted down in `test/e2e/support/custom.ts`. When these get fixes, they should be removed from the list, and at some point we can set `skipFailures = false` again.

### Steps To Reproduce

1. Try setting `skipFailures = true` in `custom.ts`, or comment out the entry for each known violation
2. Watch all the tests mentioned in the list of known violations fail

### Additional Information

If you comment out entries in `knownWcagViolations` and run each test, you'll get a link to a help page where you can investigate what needs to be done to fix the problem. Also, if you click on the error in the Cypress log, you can inspect which nodes/elements were found to be in violation.
